etc/settings/settings_test.json

```diff
@@ -217,15 +217,23 @@
                 "default": "Verilog",
                 "arg": "netlist_lang"
             },
-            "no_dsp_checkbox": {
-                "label": "No DSP Blocks",
-                "widgetType": "checkbox",
-                "arg": "no_dsp"
-            },
-            "no_bram_checkbox": {
-                "label": "No Block Ram",
-                "widgetType": "checkbox",
-                "arg": "no_bram"
+            "dsp_spinbox_ex": {
+                "label": "Dsp limit",
+                "widgetType": "Spinbox",
+                "minVal": 0,
+                "maxVal": 154,
```

Comment:
I believe this value is dependent on the device.xml so we can't hardcode it.

Reply:
Plus it will be dependent on the device user selects for a design project.

Reply:
I've put zeros.

Reply:
That hardcoded number "154" is just as an example right? The actual value needs to be read from the device.xml, the number is different for every devices.

Reply:
Also, by default is has to be set to the max range, not 0

Reply:
this value will be updated after device.xml loaded.

Reply:
> Also, by default is has to be set to the max range, not 0

@alaindargelas different devices have this value different. What max range do you mean? anyway we have another json upstream

Reply:
We reload the settings dependent on device.xml using the reloadSettings() function. This is a new feature since now we might have values in settings_test.json that is dependent on device.xml.